INN/umbrella-workdayminn
# Child theme footer

## Changes

This pull request makes the following changes:

- Creates relevant template files + styles for Workday child theme footer to match design. Does not include logos section.

![Screen Shot 2019-07-31 at 3 43 38 PM](https://user-images.githubusercontent.com/18353636/62242836-133ba900-b3aa-11e9-9acf-e874618b081c.png)

## Why



For #3 

## Testing/Questions

Features that this PR affects:

- Workday child theme footer

Steps to test this PR:

1. Add in social links to theme options
2. Verify the footer styling looks ok
